test/test_safemode_eval.rb

```diff
@@ -80,6 +80,10 @@ def test_interpolated_xstr_should_raise_security
     assert_raise_security '"#{`ls -a`}"'
   end
 
+  def test_should_not_allow_access_to_bind
+    assert_raise_security "self.bind('an arg')"
```

Comment:
Even though there is no longer `bind` method, is this possible to use `get_binding` directly instead?